devtools/procwrap/main.go

```diff
@@ -168,7 +167,9 @@ func stopWith(lock bool, sig os.Signal) int {
 	log.Println("stopping", flag.Arg(0))
 	close(stopping)
 
-	cmd.Process.Signal(sig)
+	if err := cmd.Process.Signal(sig); err != nil {
+		log.Println("ERROR: ", err)
```

Comment:
`panic` would be better here, the code below has no condition to handle if the signal failed to send, and it's assumed it will never fail.